backend/plugins/operation_api/src/modules/tasks/graphql/resolvers/utils.ts

```
@@ -0,0 +1,1130 @@
+      action: 'findWithChild',
+      input: {
+        query: { _id: { $in: departmentIds } },
+        fields: { _id: 1 },
+      },
+      defaultValue: [],
+    });
+
+    filter.departmentIds = { $in: departments.map((item) => item._id) };
+  }
+
+  if (_ids && _ids.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)